Gems/Atom/RHI/Vulkan/Code/Include/Platform/Emscripten/platform_builders_emscripten_files.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(FILES
+    Atom_RHI_Vulkan_Platform.h
```

Comment:
Will have to be moved to WebGPU, but that's on another branch https://github.com/o3de/o3de/tree/webgpu-rhi